analyzer/queries/queries.go

```diff
@@ -1147,4 +1147,107 @@ var (
     WHERE
       (evs.abi_parsed_at IS NULL OR evs.abi_parsed_at < abi_contracts.verification_info_downloaded_at)
     LIMIT $2`
+
+	RuntimeConsensusAccountTransactionStatusUpdate = `
+    -- Transaction should have been in the previous successful round.
+    -- We currently don't have the runtime-block status field in the DB, so find
+    -- the previous successful round by taking the first round which has at least one transaction.
+    WITH tx_round AS (
+      SELECT
+        MAX(rt.round) AS round
+      FROM chain.runtime_transactions AS rt
+      WHERE
+        rt.runtime = $1 AND
+        rt.round < $2
+    ),
+    matched_tx AS (
+      SELECT
+        rt.runtime,
+        rt.round,
+        rt.tx_index
+      FROM chain.runtime_transactions AS rt
+      JOIN chain.runtime_transaction_signers AS rts ON
+        rt.runtime = rts.runtime AND
+        rt.round = rts.round AND
+        rt.tx_index = rts.tx_index
+      WHERE
+        rt.runtime = $1 AND
+        rt.round = (SELECT round FROM tx_round) AND
+        rt.method = $3 AND
+        rts.signer_address = $4 AND
+        rts.signer_index = 0 AND
+        rts.nonce = $5
+      LIMIT 1
+    )
+    UPDATE chain.runtime_transactions rt
+    SET
+      success = $6,
+      error_module = $7,
+      error_code = $8,
+      error_message = $9
+    FROM matched_tx
+    WHERE
+      rt.runtime = matched_tx.runtime AND
+      rt.round = matched_tx.round AND
+      rt.tx_index = matched_tx.tx_index`
+
+	RuntimeConsensusAccountTransactionStatusUpdateFastSync = `
+    INSERT INTO todo_updates.transaction_status_updates (runtime, round, method, sender, nonce, success, error_module, error_code, error_message)
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)`
+
+	RuntimeTransactionStatusUpdatesRecompute = `
```

Comment:
This looks good since it's similar to the query above, but if you have a chance could you double check whether the fast-sync eden e2e regression test has one of the consensus-accounts txs and that it's being properly processed? It'd be nice to have another data point confirming that finalizeFastSync will work as expected. Thanks!

Reply:
Yes, for example the `consensus.Withdraw`'s which are before round 8059839 in the https://github.com/oasisprotocol/nexus/blob/main/tests/e2e_regression/eden/expected/emerald_txs_by_method.body test